pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -639,8 +639,8 @@ async def run_repomdrecord_download(name, location_href, downloader):
                     checksum_types[record.type] = record_checksum_type
                     record.checksum_type = record_checksum_type
 
-                    if self.mirror and record.location_base:
-                        raise ValueError(XML_BASE_AND_MIRROR_INCOMPATIBLE_ERR_MSG)
+                    if self.mirror and record.location_base or ".." in record.location_href:
+                        raise ValueError(MIRROR_INCOMPATIBLE_REPO_ERR_MSG)
```

Comment:
Honestly at this point we may want to reconsider the hard error, and maybe have some combination of options which say "mirror if you can, but if not, do an autopublish"

Or not, perhaps that would actually be more difficult for Katello to deal with rather than simpler.  A discussion to have.

Reply:
I think I'd prefer "you asked for mirror and can't have it, FAIL". It's a lot easier for the invoker to switch between "failed" and "completed", than it is to switch between "completed" and "completed, but not what you asked for".